ios/Portals Ecommerce/Ecommerce.entitlements

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict/>
+</plist>
```

Comment:
For whatever reason when I validated that Portals still worked with data protection entitlements, it started complaining about not having an Entitlements file, even though it was empty after I removed it 🤷‍♂️.